sdk/README.md

```diff
@@ -60,7 +60,7 @@ As an example, you can change the content of `./src/guest/src/main.rs` to:
 ```rust
 #![cfg_attr(target_arch = "riscv32", no_std, no_main)]
 
-use nexus_rt::{println, read_private_input, write_output};
+use nexus_rt::{postcard, println, read_private_input, write_output};
```

Comment:
This import of `postcard` is superfluous to this particular example. If you wanted to submit a fuller example to exist alongside this one within the quickstart and that included deserialization using `postcard` we'd be interested in potentially incorporating that.